src/supervisor.adoc

```diff
@@ -829,10 +829,59 @@ to implement that we consider it worth supporting even if only rarely
 enabled.
 ====
 
-The definition of the CBZE field is furnished by the Zicboz extension.
-
-The definitions of the CBCFE and CBIE fields are furnished by the Zicbom
-extension.
+The Zicboz extension adds the `CBZE` (Cache Block Zero instruction enable) field
+to `senvcfg`. The `CBZE` field controls execution of the cache block zero
+instruction, `CBO.ZERO`, in privilege modes U and VU.
+
+Execution of `CBO.ZERO` in U-mode is enabled only if execution of the
+instruction is enabled for use in S-mode and `CBZE` is set to 1; otherwise, an
+illegal-instruction exception is raised.
+
+Execution of `CBO.ZERO` in VU-mode is permitted only if the instruction is
+HS-qualified and execution of the instruction is enabled for use in VS-mode.
+If the instruction is not HS-qualified, an illegal-instruction exception is
+raised. If the instruction is HS-qualified, execution in VU-mode is enabled
+only if the instruction is enabled for execution in VS-mode and `CBZE` is
+set to 1; otherwise, a virtual-instruction exception is raised.
```

Comment:
The Supervisor chapter shouldn't have any mention of VS or VU modes, or of the term HS-qualified.  I think you can delete this whole paragraph.